<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat</class>
 <widget class="QWidget" name="Chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <widget class="QScrollArea" name="notificationsScroll">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>281</width>
     <height>701</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>279</width>
      <height>699</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Chats</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="chat_text_box">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>690</y>
     <width>701</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="plusButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>690</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="emojiButton">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>690</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="chat_frame">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>861</width>
     <height>631</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="ChatArea" name="active_chat">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>861</width>
      <height>671</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>690</y>
     <width>31</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="newChatBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>5</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="current_chat_icon">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Avatar</string>
   </property>
  </widget>
  <widget class="QLabel" name="curent_chat_label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>761</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current chat</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChatArea</class>
   <extends>QScrollArea</extends>
   <header>chat_area</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
